codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java

```diff
@@ -694,10 +695,22 @@ private InterfaceHttpData findMultipartDisposition() {
                             String name = cleanString(values[0]);
                             String value = values[1];
 
+                            boolean shouldDecode = false;
+                            int last = name.length() - 1;
+                            if (name.charAt(last) == '*') {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 5 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS134)